ipad/Constants/StringConstants.swift

```diff
@@ -121,7 +121,7 @@ struct WatercraftFieldHeaderConstants {
         static let failedToStop = "Watercraft pulled over for failing to stop at the inspection station"
         static let ticketIssued = "Violation ticket issued"
         static let watercraftHasDrainplugs = "Does the watercraft have a drain plug located on the lower transom? *"
-        static let drainplugRemovedAtInspection = "Was the drain plug removed at the time of inspection? *"
+        static let drainplugRemovedAtInspection = "Was the drain plug already removed when arrived at the inspection station? *"
```

Comment:
This is non blocking: 

This sentence seems clunky but maybe this was what the SO requested?

Could be:

- "Was the drain plug already removed when you arrived at the inspection station? *"
- "Was the drain plug already removed upon arrival at the inspection station? *"

Reply:
I agree with it being clunky.

I'll have a talk to confirm with Keeghan and one of the SO's to review before merging.